airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/array.sql

```diff
@@ -6,28 +6,46 @@
     - postgres: unnest() -> https://www.postgresqltutorial.com/postgresql-array/
 #}
 
-{# flatten -------------------------------------------------     #}
+{# unnest -------------------------------------------------     #}
 
 {% macro unnest(array_col) -%}
   {{ adapter.dispatch('unnest')(array_col) }}
 {%- endmacro %}
 
 {% macro default__unnest(array_col) -%}
-    unnest({{ adapter.quote_as_configured(array_col, 'identifier')|trim }})
+    unnest({{ array_col }})
 {%- endmacro %}
 
 {% macro bigquery__unnest(array_col) -%}
-    unnest({{ adapter.quote_as_configured(array_col, 'identifier')|trim }})
+    unnest({{ array_col }})
```

Comment:
laugh at me if i am not understanding this, but isn't this the same as the default? can do we need it?

Reply:
yes it's the same